alcs-frontend/src/app/features/application/applicant-info/application-details/parcel/parcel.component.ts

```diff
@@ -24,14 +28,30 @@ export class ParcelComponent implements OnInit, OnChanges {
 
   constructor(
     private applicationDocumentService: ApplicationDocumentService,
-    private parcelService: ApplicationParcelService
+    private parcelService: ApplicationParcelService,
+    private route: ActivatedRoute
   ) {}
 
   ngOnInit(): void {
     if (this.parcelType === 'other') {
       this.pageTitle = 'Other Parcels in the Community';
       this.showCertificateOfTitle = false;
     }
+
+    this.route.fragment.pipe(takeUntil(this.$destroy)).subscribe((fragment) => {
```

Comment:
Maybe you could try retrieving the id from this.route.snapshot.paramMap.get... and using afterIniti instead of onInit with setTimeout 200?

Reply:
Changed! Its no longer using a set timeout and executing in ngAfterContentChecked.